versions.txt

```diff
@@ -1,7 +1,7 @@
 # The Jaeger version to use by default. This is updated manually. Make sure to update the changelog
 # and add an upgrade procedure (pkg/upgrade) for the new version.
-jaeger=1.57.0
+jaeger=1.59.0
 
 # DO NOT EDIT the next value, it is updated automatically during the release.
 # Represents the current (latest) release of the Jaeger Operator.
-operator=1.57.0
+operator=1.59.0
```

Comment:
https://github.com/jaegertracing/jaeger-operator/blob/main/RELEASE.md#releasing 

> Change the versions.txt so that it lists the target version of the Jaeger (if it is required). Don't touch the operator version: it will be changed automatically in the next step.